### PR TITLE
Dc/feature/maintain state after refresh and storage clear

### PR DESCRIPTION
**Pull request by orangeboy55555**
Contrary to the branch name, I did not actually use the backend for the state.

### CODE REVIEW

**Review by WeiyuSun — APPROVED** [review]
looks good

File: backend/middleware/user-middleware.js [hunk 1]
```diff
@@ -152,5 +152,26 @@ function unsubscriptionValidation(req, res, next) {
     })
 }
 
-module.exports = {userIdValidation, emailValidation, passwordValidation, audienceIdValidation, creatorIdValidation, subscriptionValidation, unsubscriptionValidation}
+function setNotificationValidation(req, res, next) {
+    let creatorId = req.query.creator_id
+    let audienceId = req.query.user_id
+    let setting = req.query.set_notification;
+
+    console.log(`creator is is ${creatorId}, audience id is ${audienceId}, settings is ${setting}`)
```
Comment (ShahzaibParacha) [inline backend/middleware/user-middleware.js:160]: This too

**Review by ShahzaibParacha — APPROVED** [review]
Looks good

File: backend/middleware/user-middleware.js (hunk 1 above)
Comment (orangeboy55555) [reply backend/middleware/user-middleware.js:160]: This is the backend, so Weiyu and Da Tan are probably using it for debugging.

**Pull request merged**